strings/swap_case.py

```diff
@@ -0,0 +1,48 @@
+'''
+This algorithm helps you to swap cases.
+
+User will give input and then program will perform swap cases.
+
+In other words, convert all lowercase letters to uppercase letters and vice versa.
+For example:
+1)>>>Please input sentence: Algorithm.Python@89
+  aLGORITHM.pYTHON@89
+2)>>>Please input sentence: github.com/mayur200
+  GITHUB.COM/MAYUR200
+
+Constraints: Length of string should between 1 and 1000
+
+
+'''
+import re
+'''
+This re.compile() function saves the pattern from 'a' to 'z' and 'A' to 'Z' into 'regexp' variable
+'''
+regexp = re.compile('[^a-zA-Z]+')
+
+
+def swap_case(sentence):
+    """
+    This function will convert all lowercase letters to uppercase letters and vice versa.
+    >>>swap_case('Algorithm.Python@89')
```

Comment:
```suggestion
    >>> swap_case('Algorithm.Python@89')
```
Without the space, the test is never run.

Reply:
I will fix it

Reply:
I have added space. I have commited changes